### PR TITLE
<InputWithLabel />, <AutoCompleteWithLabel /> - added icon to error message

### PR DESCRIPTION
**Pull request by saarkuriel**
### 🔦 Summary
<InputWithLabel />, <AutoCompleteWithLabel /> - added icon to error message

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX @wuwa  
- 📚 Change is documented
  - [x] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/LabelledElement/LabelledElement.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
   align-items: center;
   position: absolute;
   pointer-events: none;
-  left: 18px;
+  left: 16px;
```
[the remark this answers is not in the document]
Comment (saarkuriel) [reply src/LabelledElement/LabelledElement.scss]: @LittleTal didn't see the correct changes. Now everything is aligned.

**Comment by saarkuriel** [comment]
> Do we have a Zeplin specification for these changes?

Yes, It's in our project. do u have permissions or should I invite you? (New Checkout responsive)

**Comment by nitayneeman** [comment]
> > Do we have a Zeplin specification for these changes?
> 
> Yes, It's in our project. do u have permissions or should I invite you? (New Checkout responsive)

I have this one:
image

But I don't see there the icon, is it the correct spec?

**Comment by saarkuriel** [comment]
> > > Do we have a Zeplin specification for these changes?
> > 
> > 
> > Yes, It's in our project. do u have permissions or should I invite you? (New Checkout responsive)
> 
> I have this one:
> image
> 
> But I don't see there the icon, is it the correct spec?

yes, our designer Nati updated the spec along with @wuwa . She will update that Zeplin

**Pull request merged**